test/parse.js

```diff
@@ -75,6 +75,11 @@ test('handle `+` correctly', t => {
 	t.deepEqual(queryString.parse('foo+faz=bar+baz++'), {'foo faz': 'bar baz  '});
 });
 
+test('additional tests to clarify exisiting number parsing behavior', t => {
```

Comment:
The test should have a description reflecting what it's testing, which is "parses numbers with exponential notation as string"

Reply:
@sindresorhus Ah right. Thanks for the suggestion. I always have a hard time naming variables and test descriptions. May my I think about it too much. Anyway, I updated with the description :+1: